### PR TITLE
Update issue templates ot use the new GH Types, and add Troubleshooting template

### PR DESCRIPTION
**Pull request by renchap**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/3.troubleshooting.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      required: false
+  - type: textarea
+    attributes:
+      label: Environment
+      description: |
+        Details about your environment, like how Mastodon is deployed, if containers are used, version numbers, etc.
+      value: |
+        Please at least include those informations:
+        - Operating system: (eg. Ubuntu 20.04)
+        - Ruby version: (from `ruby --version`, eg. v3.1.2)
```
Comment (mjankowski) [inline .github/ISSUE_TEMPLATE/3.troubleshooting.yml:64]: Not important/critical, but maybe align the various versions to current releases? (ie https://github.com/mastodon/mastodon/pull/32423)

**Review by ClearlyClaire — DISMISSED** [review]
This looks good, but what does this mean for forks, which may not have access to the GitHub issue beta?

**Comment by renchap** [comment]
> This looks good, but what does this mean for forks, which may not have access to the GitHub issue beta?

Then you probably want to ignore those changes until this gets into GA. FYI we applied to the beta and got approved after 2 days, so maybe this is also a viable alternative.

**Comment by renchap** [comment]
Updated to integrate the changes from #32423

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**